### PR TITLE
Data generator script/seeding data

### PR DESCRIPTION
**Pull request by Angelhendrix**

### CODE REVIEW

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,48 @@
+0 info it worked if it ends with ok
```
Comment (NgoMike) [inline npm-debug.log:1]: I think this log can be added to gitignore

File: seed.js [hunk 2]
```diff
@@ -1,40 +1,219 @@
...
-  // save each one into db
-  db.insertOne(eachRestaurant, (error) => {
-    if (error) {
-      throw error;
-    } else {
-      counter++;
-      if (counter === 119) {
-        mongoose.disconnect();
+// const getReviews = () => {
+//   // random length for how many reviews will render with minimum of 3 reviews
```
Comment (NgoMike) [inline seed.js]: a lot of code commented out. These can be deleted if not used

File: seed.js [hunk 3]
```diff
@@ -1,40 +1,219 @@
...
+        "Larissa", "Cyrus", "Eleanor", "Rhiannon", "Dane", "Kelly", "Jelani", "Amity", "Shannon", "Hector", "Heather", "Bethany", "Octavia", 
+        "Rahim", "Madeson", "Myra", "Macaulay", "Whoopi", "Brielle", "Gray", "Alyssa", "Naomi", "Courtney", "Kellie", "Deirdre", "Hamish", "Colton", 
+        "Logan", "Teegan", "Keith", "Leonard", "Maile", "Ivan", "Clinton", "Aiko", "Lev", "Minerva", "Nora", "Arthur", "Fallon", "Winifred", 
+        "Malachi", "Porter", "Rosalyn", "Martin", "Dara", "Alexander", "Hasad", "Bert", "Cole"
+        ]
+      var name = names[Math.floor(Math.random()*names.length)];
+      return name;
+    }
+
+    const makeRandomCity = () =>{
```
Comment (NgoMike) [inline seed.js]: nice generator functions!

File: seed@.js [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+        Array(i).fill(i).forEach(function() {
+          array.push(faker.lorem.sentence())
+        });
+      }
+      return array;
+    }
+
+    const writeStream = fs.createWriteStream('generateDataFile.json');
+
+    function make10MillData() {
```
Comment (NgoMike) [inline seed@.js:124]: thumbs up!

**Review by NgoMike — APPROVED** [review]
A lot of code that was commented out can be cleaned up but otherwise approved!

File: db/mongodb.js [hunk 5]
```diff
@@ -1,25 +1,45 @@
...
+  dinedDate: String,
+  rating: Number,
+  review: [],
 });
+  // restaurantId: Number,
+  // restaurantName: String,
+  // restaurantReviews: [
+  //   {
+  //     username: String,
+  //     city: String,
```
Comment (MichaelDurfey) [inline db/mongodb.js]: Look out for so much commented out code

File: db/mongodb.js [hunk 6]
```diff
@@ -1,25 +1,45 @@
...
+//       rating: Number,
+//       review: String
+//   });
+
 
 const Restaurant = mongoose.model('Restaurant', restaurantSchema);
 
 function insertOne(restaurant, callback) {
-  Restaurant.create(restaurant, callback);
+  Restaurant.create(restaurant, callback).exec();
```
Comment (MichaelDurfey) [inline db/mongodb.js]: You could probably take out this .exec()

File: seed@.js [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+const cities = require('./cities.js');
+const faker = require('faker');
+const fs = require('fs');
+const mongoose = require('mongoose');
+const db = require('./db/mongodb.js');
+
+const { exec } = require('child_process');
+
+
+let generateData = () => {
```
Comment (MichaelDurfey) [inline seed@.js:13]: Looks like there are two files in two different places here? Might want to get rid of one if they're identical

File: npm-debug.log (hunk 1 above)
Comment (Angelhendrix) [reply npm-debug.log:1]: i will added it now

File: seed.js (hunk 2 above)
Comment (Angelhendrix) [reply seed.js]: you got it boss

File: seed.js (hunk 3 above)
Comment (Angelhendrix) [reply seed.js]: yeeeeee

File: db/mongodb.js (hunk 5 above)
Comment (Angelhendrix) [reply db/mongodb.js]: deleting it now

File: db/mongodb.js (hunk 6 above)
Comment (Angelhendrix) [reply db/mongodb.js]: good catch its pointless

File: seed@.js (hunk 7 above)
Comment (Angelhendrix) [reply seed@.js:13]: yea I forgot to delete the old file once I got all the functionality working thanks bud

**Pull request merged**